File: app/android/java/MoviesApp/app/src/main/java/com/example/moviesapp/MainActivity.java
package com.example.moviesapp;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;

import com.example.moviesapp.view.screens.componentslist.ComponentsFragment;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction transaction = fragmentManager.beginTransaction();
        transaction.replace(
                R.id.fragmentContainer,
                new ComponentsFragment()
        );
        transaction.commit();
        setContentView(R.layout.activity_main);
    }
}

File: app/kotlin/MoviesApp/app/src/main/java/com/example/moviesapp/MainActivity.kt
package com.example.moviesapp

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.moviesapp.view.screens.componentslist.ComponentsFragment

class MainActivity: AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val transaction = supportFragmentManager.beginTransaction()
        transaction.replace(
                R.id.fragmentContainer,
                ComponentsFragment()
        )
        transaction.commit()
        setContentView(R.layout.activity_main)
    }
}